test/tests/httpTest.js

```diff
@@ -16,6 +16,16 @@ describe("Zotero.HTTP", function () {
 				}
 			}
 		);
+		httpd.registerPathHandler(
+			'/test-redirect.html',
+			{
+				handle: function (request, response) {
+					response.setHeader("Content-Type", 'text/html', false);
+					response.setStatusLine(null, 200, "OK");
+					response.write("<html><head><meta http-equiv=\"refresh\" content=\"2000; url = 'test.html'\"/></head><body></body></html>");
```

Comment:
2000 → 2 (and we should ignore anything over 15, as noted above)

I think we should also test without the spaces or quotes, since I think those are less common.